apps/etterlatte-saksbehandling-ui/client/src/components/behandling/trygdetid/Trygdetid.tsx

```diff
@@ -59,6 +60,20 @@ export const Trygdetid = ({ redigerbar, behandling, vedtaksresultat, virkningsti
 
   const visFlereTrygdetider = useFeatureEnabledMedDefault('foreldreloes', false)
 
+  const personopplysninger = usePersonopplysninger()
+
+  const mapNavn = (fnr: string): string => {
+    const opplysning = personopplysninger?.avdoede?.find(
+      (personOpplysning) => personOpplysning.opplysning.foedselsnummer === fnr
+    )?.opplysning
+
+    if (!opplysning) {
+      return fnr
+    }
+
+    return `${opplysning.fornavn} ${opplysning.etternavn} (${fnr})`
```

Comment:
```suggestion
    return `${formaterNavn(opplysning)} (${fnr})`
```
siden vi allerede har en funksjon for å formatere navn (må også importeres)

Reply:
Let etter men med feil navn - takk